dao/db.go

```diff
@@ -25,6 +25,11 @@ var (
 	conf = config.Get()
 )
 
+// Postgres error codes
+const (
+	uniqueConstraintViolation = "23505"
```

Comment:
Can we make this caps, also maybe `PG_UNIQUE_CONSTRAINT_VIOLATION` instead to indicate it's a PG error code.

Reply:
done